src/index.html

```diff
@@ -1,17 +1,164 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label>
+            Surname:
+            <input
+             name="surname"
```

Comment:
Must be two spaces offset(but now 1)
![image](https://user-images.githubusercontent.com/54936044/228380672-c08e6568-9a52-4ccc-a755-3b125dcef1d8.png)